build/x86_64_sles12/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM opensuse/leap:42.3
 
-RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_c++/SLE_12_SP3/devel:libraries:c_c++.repo \
+RUN zypper addrepo https://download.opensuse.org/repositories/Apache/SLE_12_SP4/Apache.repo \
+ && zypper addrepo https://download.opensuse.org/repositories/server:database/SLE_15/server:database.repo \
```

Comment:
Can you please add a comment explaining why we're adding a SLES 15 repo here?

Reply:
Done

Reply:
Actually, I suspect we don't really need to install `libhiredis0_13` explicitly — we will pick up `hiredis-devel` version 0.13 from this repo, which will pull in the right version of the shared library. Can we verify that and, if so, update the comment? Thanks.

Reply:
You are right. I removed the line for installing the  libhiredis0_13  and it still has the right version. I have updated the code and I will update the comment too
8e3f149c42e5:/ # zypper search libhiredis    
Loading repository data...
Warning: Repository 'NON OSS Update' appears to be outdated. Consider using a different mirror or server.
Warning: Repository 'OSS Update' appears to be outdated. Consider using a different mirror or server.
Reading installed packages...

S | Name                     | Summary                                      | Type   
--+--------------------------+----------------------------------------------+--------
  | libhiredis0_12           | Shared library for hiredis                   | package
i | libhiredis0_13           | Shared library for hiredis                   | package
  | libhiredis0_13-debuginfo | Debug information for package libhiredis0_13 | package

Reply:
Comment just updated.